### PR TITLE
Add Terminology section. Fixes #12.

### LINKED ISSUE #12: Use Cases document missing Terminology section
The Use Cases document defines and uses a number of terms, but there's no Terminology section to systematically capture the definitions introduced.  Please add it and populate it.

One term that belongs in the Terminology section is "DID registry".  Another is "DID Method", as well as "DID Controller", "DID Subject", "Relying Party", and probably "DID" itself.

### PR DESCRIPTION
**Pull request by peacekeeper**
***
<a href="https://pr-preview.s3.amazonaws.com/peacekeeper/did-use-cases/pull/33.html" title="Last updated on Nov 6, 2019, 9:11 AM UTC (b0487f7)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/did-use-cases/33/7dd8eec...peacekeeper:b0487f7.html" title="Last updated on Nov 6, 2019, 9:11 AM UTC (b0487f7)">Diff</a>

### CODE REVIEW

File: terms.html [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+describing the subject. These documents are graph-based data structures that
+are typically expressed using [[JSON-LD]], but may be expressed using other
+compatible graph-based data formats.
+  </dd>
+
+
+  <dt><dfn data-lt="">DID method</dfn></dt>
+
+  <dd>
+A definition of how a specific <a>DID scheme</a> can be implemented
```
Comment (jandrieu) [inline terms.html:47]: This definition isn't quite right. the "DID scheme" is "did", the DID method is something different, which I believe is specified within the DID as the method-string.

File: terms.html [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+on a specific <a>distributed ledger</a> or network, including the precise
+method(s) by which <a>DIDs</a> are resolved and deactivated and <a>DID
```
Comment (jandrieu) [inline terms.html:49]: We should avoid using "method" with a subtly different meaning within the definition of a DID Method. Perhaps "mechanisms by which one resolves and interacts with DIDs and DID Documents".

File: index.html [hunk 3]
```diff
@@ -83,6 +83,10 @@ <h2>Introduction</h2>
         <p>This document attempts to explain what you can do with DIDs and why you might care to use them. We start by describing the basic conceptual framework for DIDs, including basic terminology, followed by the actions one can take with a DID. Then we introduce the defining features and benefits of DIDs along with five focal use cases illustrating those features and benefits. 
         </p>
       </section>
+    <section class="informative">
+        <h1>Terminology</h1>
```
Comment (jandrieu) [inline index.html:87]: This section should be flagged as under discussion, referencing https://github.com/w3c/did-core/issues/4 at a minimum. There's a way in ReSpec to do that automagically.

File: terms.html [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+deactivation of <a>decentralized identifiers</a>.
+A DID registry is a type of verifiable data registry (see [[VC-DATA-MODEL]]).
+  </dd>
+
+  <dt><dfn data-lt="">DID resolver</dfn></dt>
+
+  <dd>
+A system that is capable of retrieving a <a>DID document</a> for a given <a>DID</a>.
+These systems are specified in the DID Resolution specification
+[[DID-RESOLUTION]].
```
Comment (jandrieu) [inline terms.html:66]: We need to figure out how to reference external documents like this. A concern was raised that this reads as if the DID Resolution specification is already completed work, while this document should be informing that work. If we define resolver simply based on the function of retrieving a DID Document, that would probably solve the concern.

**Review by jandrieu — CHANGES_REQUESTED** [review]
Still needs some work, but we'd like to get it into FPWD if we can make the adjustments.

**Comment by peacekeeper** [comment]
@jandrieu Good feedback, but note that I simply copied the terms from the did-core spec:
https://github.com/w3c/did-core/blob/master/terms.html

Are you proposing to make those changes there as well?

**Comment by jandrieu** [comment]
I would. Some of those are fundamentally wrong... like referring to the did method as the schema.

File: terms.html [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+</p>
+
+<dl class="termlist">
+
+  <dt><dfn data-lt="DID|DIDs">decentralized identifier</dfn> (DID)</dt>
+
+  <dd>
+A globally unique identifier that does
+not require a centralized registration authority because it is
+registered with <a>distributed ledger technology</a> (DLT) or other form of
```
Comment (tplooker) [inline terms.html:15]: Do we want to rephrase that anchoring a DID to a distributed ledger is optional, for example did:key did:nacl did:peer are off ledger did's?

**Comment by philarcher** [comment]
I believe everything in this PR has now been included in the current draft (see https://w3c.github.io/did-use-cases/#concepts-of-decentralized-identity). This addition has been flagged in various places as highlighted in this commentary and in the WG. I would therefore like to close this PR *without* merging it.

**Comment by jandrieu** [comment]
@peacekeeper Let us know if we can close this out. I think the approach we discussed on Tuesday is pulled into the FPWD.

If there's still some edits you'd like, we should probably do it in a separate PR.

**Comment by jandrieu** [comment]
Closing this as the FPWD has overtaken this PR with good consensus from the WG.

**Pull request closed without merging**